pkg/apis/kubeone/types.go

```diff
@@ -343,6 +343,11 @@ type KubeletConfig struct {
 	// If not provided, default value provided by kubelet will be used
 	// (max. 110 pods per node)
 	MaxPods *int32 `json:"maxPods,omitempty"`
+
+	// PodPidsLimit configures the maximum number of processes running in a Pod
```

Comment:
There should be a dot at the end of this line for better API docs rendering, i.e. `Pod.`